src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsRestClient.java

```diff
@@ -83,6 +83,9 @@ public ProjectApi name(String name) throws RestApiException {
         if (listParameter.getStart() > 0) {
             query = UrlUtils.appendToUrlQuery(query, "S=" + listParameter.getStart());
         }
+        if (listParameter.getFilterType() != ListRequest.FilterType.ALL) {
```

Comment:
I'd prefer to not execute the body in case of `null` - otherwise you need to know how `UrlUtils.appendToUrlQuery` handles `null`.

Reply:
Done :+1: